### DISCUSSION TITLE
Definately a keeper

### DISCUSSION THREAD

**Discussion by erethizon** [community/community#39705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

So far I'm really liking lists. I am keeping track of student assignment repos this way, and it is making it much easier to sift through all my forked repos. Thanks!

**Comment by ossanna16** [comment]
Hi there @erethizon and welcome to our community! Thank you for the kind feedback!

**Reply by Ismailtsafe** [reply]
Hi dear I am interested in

I am learning about it

**Comment by AQAqoma** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> So far I'm really liking lists. I am keeping track of student assignment repos this way, and it is making it much easier to sift through all my forked repos. Thanks!

**Comment by Rochebeard** [comment]
Help me,is head

**Comment by Mehedi553948** [comment]
hi

**Reply by Mehedi553948** [reply]
hi

**Comment by Rochebeard** [comment]
Yes

**Comment by Nwode** [comment]
Yes maybe

**Comment by Nwode** [comment]
What is love all about 💖👄

**Comment by Knowledge573** [comment]
Great leaning
